src/indev/lv_indev.c

```diff
@@ -1362,6 +1365,35 @@ static void indev_proc_release(lv_indev_t * indev)
     }
 }
 
+static lv_result_t indev_proc_short_click(lv_indev_t * indev)
+{
+    /*Simple short click*/
+    lv_result_t res = send_event(LV_EVENT_SHORT_CLICKED, indev_act);
+    if(res == LV_RESULT_INVALID) {
+        return res;
+    }
+
+    /*Update streak for clicks within small distance and short time*/
+    int32_t dx = indev->pointer.last_short_click_point.x - indev->pointer.act_point.x;
+    int32_t dy = indev->pointer.last_short_click_point.y - indev->pointer.act_point.y;
+    if(dx * dx + dy * dy > indev->scroll_limit * indev->scroll_limit ||
+       lv_tick_elaps(indev->pointer.last_short_click_timestamp) > indev->long_press_time) {
+        indev->pointer.short_click_streak = 0;
+    }
+    else {
+        indev->pointer.short_click_streak = (indev->pointer.short_click_streak + 1) % 3;
```

Comment:
Saturate such that triple click is fired repeatedly, or such that no further click streak events are fired?

Reply:
I think no further click streak events should be fired.

Reply:
On PC it's restarted: 
![clicks](https://github.com/lvgl/lvgl/assets/7599318/219bc5bc-849a-46ac-8eab-f584c7f44117)

Reply:
On PC, it could be a value to notify application click counts, instead of resending the single click event.

Resend the single, double click is not what actually happens, thus I think we should ignore futhure clicks for now.

Here's a reference to what web will do 
https://stackoverflow.com/questions/6480060/how-do-i-listen-for-triple-clicks-in-javascript

Reply:
It would be really the most flexible if we incremented `short_click_streak` infinitely and added an API to get its count. As the `CLICKED` event is sent [after incrementing](https://github.com/lvgl/lvgl/pull/6187/files#diff-35a05ebbf5595318ed115bcabc7f70e6ae0b4f1f3ace3629e154a7434fe680f1R818-R821) the user can do any custom thing there.

Reply:
Added `lv_indev_get_short_click_streak()` that can be used from short clicked handlers, and kept the cycling for the simple single/double/triple click events.